src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -27,4 +29,36 @@ public UserResponseDto findByEmail(@RequestParam String email) {
                 () -> new RuntimeException("User with email " + email + " not found"));
         return userResponseDtoMapper.mapToDto(user);
     }
+
+    @GetMapping("/")
+    public List<UserResponseDto> getAll() {
```

Comment:
let's add this endpoint into SecurityConfig (I believe only admin should be able to view all users)

Reply:
added - row 40 in SC